packages/dds/sequence/src/test/memory/sharedSequence.spec.ts

```diff
@@ -28,16 +28,18 @@ describe("SharedSequence memory usage", () => {
     const numbersOfEntriesForTests = [100, 1000, 10_000];
 
     numbersOfEntriesForTests.forEach((x) => {
-        benchmarkMemory({
-            title: `Append and remove ${x} subsequences`,
-            benchmarkFn: async () => {
-                const segment = new SubSequence<number>([]);
+        benchmarkMemory(new class implements IMemoryTestObject {
+            title = `Append and remove ${x} subsequences`;
+            private segment = new SubSequence<number>([]);
+
+            async run() {
+                this.segment = new SubSequence<number>([]);
```

Comment:
Let's move this line to a `beforeIteration()` method, so the actual test is only about the `append()` and `removeRange()` calls, to make it more targeted. And create a separate test that only does the instantiation inside `run()`, nothing else, similar to the "Create empty SharedString/Matrix/map".